Automattic/jetpack
# Add map_block_settings global to jetpack-mu-wpcom

## Proposed changes:
This PR adds the  map_block_settings global to jetpack-mu-wpcom

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:
- Checkout this PR
- Follow instructions here: PCYsg-Osp-p2
- Add a new post/page
- Open devtools & make sure the iframe (post-new.php) is selected
- Output `Jetpack_Maps`:

<img width="237" alt="CleanShot 2023-04-26 at 08 40 18@2x" src="https://user-images.githubusercontent.com/528287/234490722-e4df48c9-9a31-4f29-ac3b-8c79ae76d328.png">

(This will say mapbox in your testing)
